src/shared-api-objects/user-has-access-react.component.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { getCurrentUser, userHasAccess } from "./current-user";
+
+export default function UserHasAccessReact(props) {
+  const [user, setUser] = React.useState(null);
+
+  React.useEffect(() => {
+    const subscription = getCurrentUser({ includeAuthStatus: false }).subscribe(
+      x => {
+        setUser(x);
+        console.log(x);
```

Comment:
We should not push console.log statements to github that were used in debugging.